Unrecognized field "parameterNames" Error

When I use "cmdClient.devices()", error occurs as below.

**javax.ws.rs.client.ResponseProcessingException: javax.ws.rs.ProcessingException: org.codehaus.jackson.map.exc.UnrecognizedPropertyException: Unrecognized field "parameterNames" (Class org.edgexfoundry.domain.Put), not marked as ignorable
 at [Source: org.apache.http.conn.EofSensorInputStream@35fc037d; line: 1, column: 1378] (through reference chain: org.edgexfoundry.domain.CommandDevice["commands"]->org.edgexfoundry.domain.Command["put"]->org.edgexfoundry.domain.Put["parameterNames"])**
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.extractResult(ClientInvocation.java:141)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.extractors.BodyEntityExtractor.extractEntity(BodyEntityExtractor.java:59)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientInvoker.invoke(ClientInvoker.java:104)
	at org.jboss.resteasy.client.jaxrs.internal.proxy.ClientProxy.invoke(ClientProxy.java:64)
	at com.sun.proxy.$Proxy88.devices(Unknown Source)
	at org.edgexfoundry.controller.impl.CmdClientImpl.devices(CmdClientImpl.java:44)
	at org.edgexfoundry.pkg.command_test.EdgeCommand.getAllCommandDevices(EdgeCommand.java:28)
	at org.edgexfoundry.pkg.command_test.Starter.run(Starter.java:47)
Caused by: javax.ws.rs.ProcessingException: org.codehaus.jackson.map.exc.UnrecognizedPropertyException: Unrecognized field "parameterNames" (Class org.edgexfoundry.domain.Put), not marked as ignorable
 at [Source: org.apache.http.conn.EofSensorInputStream@35fc037d; line: 1, column: 1378] (through reference chain: org.edgexfoundry.domain.CommandDevice["commands"]->org.edgexfoundry.domain.Command["put"]->org.edgexfoundry.domain.Put["parameterNames"])
	at org.jboss.resteasy.client.jaxrs.internal.ClientResponse.readFrom(ClientResponse.java:285)
	at org.jboss.resteasy.client.jaxrs.internal.ClientResponse.readEntity(ClientResponse.java:181)
	at org.jboss.resteasy.specimpl.BuiltResponse.readEntity(BuiltResponse.java:213)
	at org.jboss.resteasy.client.jaxrs.internal.ClientInvocation.extractResult(ClientInvocation.java:105)
	... 7 more
Caused by: org.codehaus.jackson.map.exc.UnrecognizedPropertyException: Unrecognized field "parameterNames" (Class org.edgexfoundry.domain.Put), not marked as ignorable
 at [Source: org.apache.http.conn.EofSensorInputStream@35fc037d; line: 1, column: 1378] (through reference chain: org.edgexfoundry.domain.CommandDevice["commands"]->org.edgexfoundry.domain.Command["put"]->org.edgexfoundry.domain.Put["parameterNames"])
	at org.codehaus.jackson.map.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:53)
	at org.codehaus.jackson.map.deser.StdDeserializationContext.unknownFieldException(StdDeserializationContext.java:267)
	at org.codehaus.jackson.map.deser.std.StdDeserializer.reportUnknownProperty(StdDeserializer.java:673)
	at org.codehaus.jackson.map.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:659)
	at org.codehaus.jackson.map.deser.BeanDeserializer.handleUnknownProperty(BeanDeserializer.java:1365)
	at org.codehaus.jackson.map.deser.BeanDeserializer._handleUnknown(BeanDeserializer.java:725)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:703)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
	at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:217)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:194)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:30)
	at org.codehaus.jackson.map.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:299)
	at org.codehaus.jackson.map.deser.SettableBeanProperty$MethodProperty.deserializeAndSet(SettableBeanProperty.java:414)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:697)
	at org.codehaus.jackson.map.deser.BeanDeserializer.deserialize(BeanDeserializer.java:580)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:217)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:194)
	at org.codehaus.jackson.map.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:30)
	at org.codehaus.jackson.map.ObjectMapper._readValue(ObjectMapper.java:2704)
	at org.codehaus.jackson.map.ObjectMapper.readValue(ObjectMapper.java:1315)
	at org.codehaus.jackson.jaxrs.JacksonJsonProvider.readFrom(JacksonJsonProvider.java:419)
	at org.jboss.resteasy.core.interception.AbstractReaderInterceptorContext.readFrom(AbstractReaderInterceptorContext.java:61)
	at org.jboss.resteasy.core.interception.AbstractReaderInterceptorContext.proceed(AbstractReaderInterceptorContext.java:53)
	at org.jboss.resteasy.plugins.interceptors.encoding.GZIPDecodingInterceptor.aroundReadFrom(GZIPDecodingInterceptor.java:59)
	at org.jboss.resteasy.core.interception.AbstractReaderInterceptorContext.proceed(AbstractReaderInterceptorContext.java:55)
	at org.jboss.resteasy.client.jaxrs.internal.ClientResponse.readFrom(ClientResponse.java:251)
	... 10 more

I think there is a problem with "parameternames" in "core-command-client / src / main / java / org / edgexfoundry / domain / Put.java"
So, I think "parameternames" should be changed to "parameterNames".

--- a/src/main/java/org/edgexfoundry/domain/Put.java
+++ b/src/main/java/org/edgexfoundry/domain/Put.java
@@ -22,14 +22,14 @@ import java.util.List;
 
 public class Put extends Action {
 
-  private List<String> parameternames;
+  private List<String> parameterNames;
 
-  public List<String> getParameternames() {
-    return parameternames;
+  public List<String> getParameterNames() {
+    return parameterNames;
   }
 
-  public void setParameternames(List<String> parameternames) {
-    this.parameternames = parameternames;
+  public void setParameterNames(List<String> parameterNames) {
+    this.parameterNames = parameterNames;
   }
 
 }

